### ISSUE TITLE
Some real tests

### ISSUE THREAD

**Issue by casevictor** [tonylukasavage/ti-mocha#23]
Tony, first of all thanks for the great work, keep improving it.
I'm novice at unit tests, I will be glad if you can add some "real" tests inside the documentation.
`E.g : Some test using logic | controllers.`
Perhaps, I didn't understand the logic about this, but in your docs only have test over ui elements.

**Comment by tonylukasavage** [comment]
I'm not entirely certain what you mean by _real_ tests, as there's fully functional tests in the docs. Perhaps you could provide a specific example of something you'd like documented?

**Comment by casevictor** [comment]
Yes for sure, maybe some examples using httpclient with callbacks, I need to mock it?
Another point that I was stuck.. how I can implement that tests wholely separate from app code?

```
It depends how and what you're testing. It is definitely possible to have your test code wholely separate from your app code.
```

Your response to.
http://blog.divhide.com/2014/06/22/ti-mocha-example/

And if you coded an App, covered by ti-mocha, please refer to us :)
